sky/cli.py

```diff
@@ -2074,6 +2073,10 @@ def logs(
     job_id = None
     if job_ids:
         job_id = job_ids[0]
+        if not job_id.isdigit():
+            raise click.UsageError(f'Invalid job ID {job_id}. '
+                                   'Job ID must be integers.')
```

Comment:
Should we check all the `job_id` to be digit?
```suggestion
        if not all(job_id.isdigit() for job_id in job_ids):
            raise click.UsageError(f'Invalid job ID {", ".join(job_ids)}. '
                                   'Job ID must be integers.')
```

Reply:
We already checked `job_ids is None or len(job_ids) <= 1, job_ids` above, but I think it will benefit for the future. Thanks!